src/test/scala/akka/persistence/cassandra/journal/MultiPluginSpec.scala

```diff
@@ -105,6 +105,8 @@ class MultiPluginSpec
 
     session.execute(s"CREATE KEYSPACE IF NOT EXISTS $journalKeyspace WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 }")
     session.execute(s"CREATE KEYSPACE IF NOT EXISTS $snapshotKeyspace WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 }")
+
+    CassandraLifecycle.awaitPersistenceInit(system, "cassandra-journal-a", "cassandra-snapshot-c")
```

Comment:
looking at the config in this test there are more journals and snapshot-stores, shall we await all of them here?
```
CassandraLifecycle.awaitPersistenceInit(system)
CassandraLifecycle.awaitPersistenceInit(system, "cassandra-journal-a")
CassandraLifecycle.awaitPersistenceInit(system, "cassandra-journal-b")
CassandraLifecycle.awaitPersistenceInit(system, "cassandra-journal-c", "cassandra-snapshot-c")
CassandraLifecycle.awaitPersistenceInit(system, "cassandra-journal-d", "cassandra-snapshot-d")
```